src/js/IAMClient.js

```diff
@@ -1,11 +1,24 @@
 // @noflow
 
-import type { AppState } from '../types/state';
+import type { AppState, AssumeRoleParams, OIDCState } from '../types/state';
 import type { Credentials } from '../types/zenko';
 import IAM from 'aws-sdk/clients/iam';
 import type { IAMClient as IAMClientInterface } from '../types/iam';
 import { getClients } from '../react/utils/actions';
 
+export const roleName = 'scality-internal/storage-manager-role';
```

Comment:
I think we should rename this to rolePath?
Or specify, the internal path it `scality-internal` and `storage-manager-role` is the name.

Reply:
I will rename it `rolePathName` as we have both.

Reply:
Done: https://github.com/scality/zenko-ui/commit/0aa0982448282802472adc30314a3164335a4b0f#diff-524d780d05f66cc8206f06f1bb1dd913e4932608ba6d60a6be5cf1f746113ba7R9